VIDEO_4_SCRIPT.md

```diff
@@ -0,0 +1,44 @@
+# Video 4: AsyncAPI and OpenAPI
+
+Hello again! Welcome to a new video about AsyncAPI. This video will discuss what AsyncAPI and OpenAPI have in common and how they differ. 
+
+Ready or not, here we go! 
+
+## What is OpenAPI?
+
+OpenAPI is a standard for describing application programming interfaces (APIs). This specification defines an open, vendor-independent description format for API services. In particular, OpenAPI can describe, develop, test, and document REST-compliant APIs.
+
+The current OpenAPI specification grew out of the predecessor project, Swagger. The development company SmartBear subjected the existing Swagger specification to an open license, handing over both maintenance and development of the initiative to the Linux Foundation. In addition to SmartBear, members of the OpenAPI initiative include industry giants such as Google, IBM, and Microsoft.
+
+## Similarities between AsyncAPI and OpenAPI
+
+AsyncAPI's ultimate goal is to work with EDA architectures as simply as with Rest APIs. The AsyncAPI specification in the asynchronous world is analogous to OpenAPI in the synchronous world. 
+
+Both initiatives share numerous similarities:
+
+- Contract-first philosophy
+- Implementation-agnostic
+- Similar syntax
+- A tooling ecosystem 
+- Extensible
+- They protect information consumption using different security schemes. (Example: user and password, API keys, certificates, OAuth, OIDC, etc.)
+
+## Differences between them
+
+ As we have seen, the two initiatives are significantly similar, but they also cover significantly different fields. REST APIs have a specific function that requires their methods and documentation approaches. REST APIs have, among others, a great solution in the form of OpenAPI, a set of standardized documentation methods for enabling tools and interactivity.
+ 
+ It's essential to understand that while the two solutions have a lot in common, they are also wildly divergent. Both OpenAPI and AsyncAPI allow for more significant cooperation and collaboration between different documented APIs. Still, OpenAPI is a solution for REST, and AsyncAPI is a solution for message-based architectures. Each aims to serve a different domain.
+
+## AsyncAPI and OpenAPI comparison
+
+As you can see, you'll find lots of similarities between OpenAPI and AsyncAPI!
```

Comment:
this sentence suggests that I should see a kind of similarities, but I see screen shot from landing pages of both initiatives, is it intended? Maybe we should use some of the images from @jessemenning in this video -> https://www.asyncapi.com/blog/openapi-vs-asyncapi-burning-questions